backend/migrations/cmd/backfill-session-properties-clickhouse/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"os"
+
+	kafka_queue "github.com/highlight-run/highlight/backend/kafka-queue"
+	"github.com/highlight-run/highlight/backend/model"
+	public "github.com/highlight-run/highlight/backend/public-graph/graph"
+	"github.com/highlight-run/highlight/backend/redis"
+	"github.com/highlight-run/highlight/backend/store"
+	"github.com/openlyinc/pointy"
+	"github.com/samber/lo"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const SessionsMaxRowsPostgres = 500
+
+func main() {
+	ctx := context.Background()
+
+	db, err := model.SetupDB(ctx, os.Getenv("PSQL_DB"))
+	if err != nil {
+		log.WithContext(ctx).Fatalf("error setting up db: %+v", err)
+	}
+
+	log.WithContext(ctx).Info("set up clients")
+
+	kafkaDataSyncProducer := kafka_queue.New(ctx,
+		kafka_queue.GetTopic(kafka_queue.GetTopicOptions{Type: kafka_queue.TopicTypeDataSync}),
+		kafka_queue.Producer, &kafka_queue.ConfigOverride{Async: pointy.Bool(true)})
+
+	redisClient := redis.NewClient()
+
+	publicResolver := &public.Resolver{
+		DB:            db,
+		DataSyncQueue: kafkaDataSyncProducer,
+		Store:         store.NewStore(db, redisClient, nil, nil, kafkaDataSyncProducer),
+	}
+
+	var sessionIds []int
+	if err := db.Raw(`
+		SELECT id
+		FROM sessions
+		WHERE created_at >= now() - interval '5 days'
+	`).Scan(&sessionIds).Error; err != nil {
+		log.WithContext(ctx).Fatal(err)
+	}
```

Comment:
should this `FindInBatches` to avoid loading the massive int array in memory? guessing we'd ultimately want to do this for all sessions

Reply:
yeah that would probably be better, but I've used this script on 3 months of sessionids / errorids before and it's worked fine. This script is fine to run on just 5 days worth of sessions because these fields were being indexed prior to turning off opensearch writes (because this code was removed)